### PR TITLE
Development ergonomics

### PR DESCRIPTION
**Pull request by pshc**
- Deleted derived `calculator.rs` to avoid noise and conflicts
- Output now includes parse error detail

### CODE REVIEW

**Comment by pshc** [comment]
Hmm, I think this allows calculator.rs to be quietly out-of-date when switching git branches...

**Comment by tcr** [comment]
This is a good commit 👍 I'm gonna borrow this refactor for my other language project too!

File: parser/build.rs [hunk 1]
```diff
@@ -1,16 +1,6 @@
...
-    match (fs::metadata("./src/calculator.lalrpop"), fs::metadata("./src/calculator.rs")) {
-        (Ok(a), Ok(b)) => {
-            if a.modified().unwrap() < b.modified().unwrap() {
-                return;
-            }
-        }
-        _ => {}
-    }
-
+    println!("cargo:rerun-if-changed=src/calculator.lalrpop");
```
Comment (tcr) [inline parser/build.rs]: This is a great feature!

Comment (pshc) [inline parser/build.rs]: Cargo is so good.

**Pull request merged**